### PR TITLE
clashapi responses and first get

### PR DESCRIPTION
**Pull request by DNSJambon**

### CODE REVIEW

**Comment by DNSJambon** [comment]
ktlint format OK

**Comment by DNSJambon** [comment]
command works but checks fails

File: riot-api/src/main/kotlin/com.noahkohrs.riot.api/account/Responses.kt
Comment (DNSJambon) [inline riot-api/src/main/kotlin/com.noahkohrs.riot.api/account/Responses.kt:1]: is it ok?

File: riot-api/src/main/kotlin/com.noahkohrs.riot.api/lol/clash/ClashApi.kt [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        // GET /lol/clash/v1/tournaments/{tournamentId}
+        @RequestLine("GET /lol/clash/v1/tournaments/{tournamentId}")
+        fun getTournamentById(
+            @Param("tournamentId") tournamentId: Int,
+        ): TournamentDto
+    }
+}
+
+// TODO: Generate a test for this class by right-clicked on <ClassName> -> Generate -> Test... -> OK
```
Comment (noahkohrs) [inline riot-api/src/main/kotlin/com.noahkohrs.riot.api/lol/clash/ClashApi.kt:94]: nit: Should be removed.

File: riot-api/src/main/kotlin/com.noahkohrs.riot.api/account/Responses.kt
Comment (noahkohrs) [reply riot-api/src/main/kotlin/com.noahkohrs.riot.api/account/Responses.kt:1]: I would rather removing the account test than exposing the constructor

**Review by noahkohrs — CHANGES_REQUESTED** [review]

**Comment by noahkohrs** [comment]
Also mark the endpoint as done in the `README.md` checklist 🫡​

**Comment by DNSJambon** [comment]
not ready, probelm

**Pull request merged**